setuptools/package_index.py

```diff
@@ -560,10 +560,7 @@ def not_found_in_index(self, requirement):
         if self[requirement.key]:  # we've seen at least one distro
             meth, msg = self.info, "Couldn't retrieve index page for %r"
         else:  # no distros seen for this name, might be misspelled
-            meth, msg = (
-                self.warn,
-                "Couldn't find index page for %r (maybe misspelled?)",
-            )
+            meth, msg = self.warn, "Couldn't find index page for %r (maybe misspelled?)"
```

Comment:
Not strictly related to `unpacked-list-comprehension (UP027)`, but I found this whilst looking for `,.*? = \(` so I figured I might make it cleaner too whilst I saw it.